docs/source/changelog.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking changes
 
+- Support for use against k8s 1.20-1.22 is no longer maintained, please upgrade
+  to k8s 1.23+ to ensure function.
```

Comment:
```suggestion
- Support for K8s 1.20-1.22 is no longer maintained, please upgrade
  to k8s 1.23+ to ensure continued working.
```

Reply:
Continued working, sounds weird, is this an acceptable sentence?

Reply:
Another option, "Versions of K8s older than 1.23 are no longer supported, they may work but this is not guaranteed"?
.
Anyway, go ahead and merge whatever wording you prefer.